packages/expo-cli/src/commands/eas-build/utils/expoUpdates/base.ts

```diff
@@ -0,0 +1,15 @@
+import { ExpoConfig, getPackageJson } from '@expo/config';
```

Comment:
Maybe rename this file to `common.ts`. It won't be perfect but it sounds better than `base.ts`.